CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Deprecated
 ### Removed
 ### Fixed
+- Fix mapping char_filter when mapping a hashtag ([#7591](https://github.com/opensearch-project/OpenSearch/pull/7591))
```

Comment:
```suggestion
- Fixed mapping char_filter when mapping a hashtag ([#7591](https://github.com/opensearch-project/OpenSearch/pull/7591))
```

Reply:
I think the entry here makes sense since we tend to follow the imperative mood - https://github.com/dangoslen/dependabot-changelog-helper/pull/131